src/wallet/rpcwallet.cpp

```diff
@@ -2911,6 +2911,7 @@ UniValue z_getnewaddress(const UniValue& params, bool fHelp)
             "z_getnewaddress ( type )\n"
             "\nReturns a new shielded address for receiving payments.\n"
             "\nWith no arguments, returns a Sapling address.\n"
+            "Generating a " + ADDR_TYPE_SPROUT + " is not allowed after Canopy has activated.\n"
```

Comment:
```suggestion
            "Generating a Sprout address is not allowed after Canopy has activated.\n"
```

The existing code will say "Generating a sprout is not allowed..."